src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -14,9 +14,49 @@
  * Controller for a help page
  */
 public class HelpWindow extends UiPart<Stage> {
+    public static final String SINGLE_NEWLINE = "\n";
+    public static final String DOUBLE_NEWLINE = "\n\n";
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    public static final String USERGUIDE_URL = "https://ay2324s1-cs2103t-w09-1.github.io/tp/UserGuide.html";
+
+    public static final String ADD_COMMAND_GUIDE =
+            "1. ADD" + SINGLE_NEWLINE
+            + "Adds a contact to the address book." + DOUBLE_NEWLINE
+            + "Format: add n/NAME p/PHONE_NUMBER e/EMAIL o/OCCUPATION a/ADDRESS [t/TAG]..." + DOUBLE_NEWLINE
+            + "Example: add n/John Doe, p/+65 98765432, e/johnd@example.com, o/Software Engineer, "
+            + "ad/John Street, Block 123, 138123, t/New Client";
```

Comment:
for the address, it should be a/, not ad/

Reply:
Thank you, Bhanuka for checking my PR. I am going to change it.